mod/01_rating/spec/set/type/metric_value_spec.rb

```diff
@@ -111,10 +111,36 @@
     end
   end
   describe "update metric value's value" do
-    it "updates metric value' value correctly" do
-      @metric_value.update_attributes! :subcards=>{"+value"=>"if nobody hates you, you're doing something wrong."}
+    it "updates metric value's value correctly" do
+      quote = "if nobody hates you, you're doing something wrong."
+      @metric_value.update_attributes! subcards: {
+        '+value' => quote
+      }
       metric_values_value_card = Card["#{@metric_value.name}+value"]
-      expect(metric_values_value_card.content).to eq("if nobody hates you, you're doing something wrong.")
+      expect(metric_values_value_card.content).to eq(quote)
+    end
+    context "with another source" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.